react/features/filmstrip/components/web/RaisedHandIndicator.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import BaseIndicator from './BaseIndicator';
+
+/**
+ * Thumbnail badge showing that the participant would like to speak.
+ *
+ * @extends BaseIndicator
+ */
+class RaisedHandIndicator extends BaseIndicator {
+    /**
+     * {@code RaisedHandIndicator} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The font-size for the icon.
+         *
+         * @type {number}
+         */
+        iconSize: React.PropTypes.number
```

Comment:
To close the loop on this: I was confused and thought the suggestion was to pass in a function but the suggestion was calling the global itself from render(). Also, there was confusion as to why passing in is needed at all and it's because on remote video add or remove the thumbnails are resized in horizontal filmstrip mode and the icons are resized.